LayoutTests/http/tests/cookies/resources/cookie-utilities.js

```diff
@@ -12,7 +12,7 @@ function createCookie(name, value, additionalProperties)
     for (let propertyName in additionalProperties) {
         cookie += `; ${propertyName}`;
         let propertyValue = additionalProperties[propertyName];
-        if (propertyValue != undefined)
+        if (propertyValue !== undefined)
```

Comment:
Typically a library uses ` == undefined` when it wants to check either `null` or `undefined`.

* `null == undefined` => `true`
* `undefined == undefined` => `true`

I don't know anything about this test code, but I wonder:

* Would it make sense for createCookie to allow a cookie value of `null` here?
* Likewise for usages below, I wonder if the library wants to remain resilient if the JSON serialized values from `getCookies` included a `null` value.

Just food for thought. I don't think anything about the change is wrong, just want to make sure some thought was given to the change in behavior to `null` values because I didn't see that described in the commit message.

Reply:
Thank you for bringing up the point. I think we can write instead to be more specific: if (value === undefined || value === null) and if (propertyValue !== undefined && propertyValue !== null).